components/Storyblok/SbHomepageMVP/SbHomepageMvp.tsx

```diff
@@ -39,33 +44,29 @@ export const SbHomepageMvp = ({
     ankle,
     mastheadPicker,
   },
-  blok,
 }: SbHomepageMvpProps) => {
   const ChangemakerCards = !!getNumBloks(changemakerCards) ? <CreateBloks blokSection={changemakerCards} /> : undefined;
   const ChangemakerCta = !!getNumBloks(changemakerCta) ? <CreateBloks blokSection={changemakerCta} /> : undefined;
 
   return (
-    <div {...storyblokEditable(blok)}>
```

Comment:
Get rid of the wrapper because we don't need the `storyblokEditable(blok)` for page components